src/cluster/setup.rst

```diff
@@ -201,7 +203,14 @@ setup the node:
 
 .. code-block:: bash
 
-     curl -X POST -H "Content-Type: application/json" http://admin:password@127.0.0.1:5984/_cluster_setup -d '{"action": "enable_cluster", "bind_address":"0.0.0.0", "username": "admin", "password":"password"}'
+     curl -X POST -H "Content-Type: application/json" http://admin:password@127.0.0.1:5984/_cluster_setup -d '{"action": "enable_cluster", "bind_address":"0.0.0.0", "username": "admin", "password":"password", "node_count":"3"}'
+
+.. warning::
+    The ``node_count`` parameter is used to set the ``[cluster]`` ``n`` value.
+    Even when joining more than 3 nodes into a cluster using this endpoint,
+    it is **strongly** recommended not to set this value higher than 3, as
+    it will directly impact the number of replicas of each document in the
+    cluster.
```

Comment:
Agree :100:%. Please merge apache/couchdb-setup#14 and I'll remove the warning here.